spec/models/draw_spec.rb

```diff
@@ -44,6 +44,17 @@
     end
   end
 
+  describe '#group_sizes' do
+    let(:draw) { FactoryGirl.build_stubbed(:draw) }
+    it 'returns an array of group sizes in the draw' do
+      instance_spy('GroupSizesQuery', call: [1, 2]).tap do |q|
+        allow(GroupSizesQuery).to receive(:new).with(draw.suites.available)
+          .and_return(q)
+      end
+      expect(draw.group_sizes).to match_array([1, 2])
```

Comment:
We can just assert that `GroupSizesQuery` gets called, we don't need to test against return values.